src/go/rpk/pkg/cli/cluster/partitions/show.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2023 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package partitions
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/adminapi"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/twmb/types"
+)
+
+func newShowPartitionAssignmentsCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	type result struct {
+		Topic     string
+		Partition int
+		Leader    int
+		NodeCore  []string
+	}
+	var partitions []int
+	cmd := &cobra.Command{
+		Use:   "show",
```

Comment:
Should this be describe, to mirror `rpk topic describe` and `rpk group describe`?

Reply:
SGTM. Fixed in https://github.com/redpanda-data/redpanda/pull/14025/commits/75b27f460ee265187d46ff102c76d196e2665212